magento/magento2-page-builder
# Bug fixed when using the decimal number in slidesToShow.

### Description (*)
Slick slider unofficially enables the use of decimal point in the slidesToShow attribute. When we define the 5.2 to the slidesToShow attribute, slick slider will show the 5 items and the 20 percent of 6 item. But we need to use undecimal number in the slidesToScroll parameter. Otherwise, the slider will miscalculate. This pull request aims to solve this problem.

![image](https://user-images.githubusercontent.com/10761479/221134715-e759b0c4-a14f-4cd1-a9fe-14afd93f6ed1.png)

### Manual testing scenarios (*)
1. Set the default slidesToShow variable 4.5 in the view.xml (https://developer.adobe.com/commerce/frontend-core/page-builder/viewports/use-breakpoints/)
2. See the products slider will show the 4 product items and half of 5 product item.
3. Make sure it looks right when the slider moves to the next element.

### Checklist
